content/eng/central/commerce_and_industries/AIC.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Atal Incubation Centres"
+date: 2019-05-20T12:00:08+05:30
+draft: true
+department: "Commerce and Industry"
+type: "post"
+---
+
+**The NITI Aayog which formed in 2015 is a policy think tank of the Government of India, established with the aim to achieve **Sustainable Development Goals** and to enhance **cooperative federalism** by fostering the involvement of **State Governments of India** in the economic policy-making process using bottom-up approach. It's initiative include "15 year roadmap", "7-year vision, strategy and action plan", **AMRUT**, **Digital India**, **Atal Innovation Mission**, Medical Education Reforms etc**.
```

Comment:
should we include definition of words like federalism, so that when people click or hover, they can understand the words

Reply:
It doesn't look good.

Reply:
Meaning is too lengthy